### ISSUE TITLE
Could you please add the timestamps to Altitude or GroundTruth structs?

### ISSUE THREAD

**Issue by ViktorYastrebov** [mavlink/MAVSDK#2391]
Is it possible to add timestamp from the 
* mavlink_hil_state_quaternion_t 
* mavlink_altitude_t 
original messages?

**Comment by julianoes** [comment]
I agree timestamps should be there and is an omission. In an idea world I would like to have proper timesync first and then timestamp everything int UTC but just the raw timestamps is probably a required first step.
